base_station/gui/README.md

```diff
@@ -0,0 +1,97 @@
+# GUI LCM Documentation 
+
+### Base Station GUI LCMs
+
+#### All GUIs:
+Camera 
+"/camera_servos"
+File: CameraServos.lcm
+Subscritions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: 
+
+DebugMessage
+"/debug_message"
+File: DebugMessage.lcm
+Subscritions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: Sends an error message
+
+Drive Velocity
+"/drive_vel_data"
+File: DriveVelData.lcm
+Subscriptions: DriveVelDataH.vue, DriveVelDataV.vue
```

Comment:
They both display the same data except DataH displays horizontally and DataV displays vertically. I think we decided on keeping both of them just in case we need to display one horizontally or vertically for spacing purposes.

Reply:
gotcha, can this be documented in the readme?